test/powershell/Modules/Microsoft.PowerShell.Utility/Group-Object.Tests.ps1

```diff
@@ -92,16 +91,65 @@ Describe "Group-Object" -Tags "CI" {
     }
 
     It "Should be able to access when output as hash table" {
-	$actual = $testObject | Group-Object -AsHashTable
+        $actual = $testObject | Group-Object -AsHashTable
 
-	$actual.Keys | Should Not BeNullOrEmpty
+        $actual.Keys | Should Not BeNullOrEmpty
     }
 
     It "Should throw when attempting to use AsString without AsHashTable" {
-	{ $testObject | Group-Object -AsString } | Should Throw
+        { $testObject | Group-Object -AsString } | Should Throw
     }
 
     It "Should not throw error when using AsString when the AsHashTable was added" {
-	{ $testObject | Group-Object -AsHashTable -AsString } | Should Not Throw
+        { $testObject | Group-Object -AsHashTable -AsString } | Should Not Throw
+    }
+}
+
+Describe "Check 'Culture' parameter in order object cmdlets (Group-Object, Sort-Object, Compare-Object)" -Tags "CI" {
+
+    BeforeAll {
+
+        $testObject = Get-ChildItem
+
+    }
+
+    It "Should accept a culture by name" {
+
+        if ( (Get-Culture).Name -eq "ru-Ru" ) {
+            $testCulture = "ru-UA"
+        }
+        else
+        {
+            $testCulture = "ru-RU"
+        }
```

Comment:
nitpick: you are mixing the open parenthesis styles (here and also the following 2 It block) :) maybe unify them as:
```
if (...) {
   ...
} else {
   ...
}
```

But this doesn't block merging the PR.